web/web.go

```diff
@@ -240,9 +241,11 @@ func MustBuildServerURL(pathPrefix string) string {
 	if err != nil {
 		panic(err)
 	}
-	hostname, err := os.Hostname()
-	if err != nil {
-		panic(err)
+	if hostname == "" {
```

Comment:
oops... sorry, I had problems with my corporate proxy to fetch dependencies... "what could possibly go wrong".